medusa/post_processor.py

```diff
@@ -1272,7 +1272,10 @@ def process(self):
                 existing_release_names.append(self.release_name or 'N/A')
                 app.RECENTLY_POSTPROCESSED[self.info_hash] = existing_release_names
             else:
-                logger.log(u'Unable to get info to move torrent later as no info hash available for: {0}'.format
-                           (self.file_path), logger.WARNING)
-
+                if not self.in_history:
+                    logger.log(u"Please consider manually move torrent to seed folder as it wasn't snatched from "
```

Comment:
Single quotes

Reply:
notice the `s it wasn't sn`
In this cases we use double quotes